golang/cosmos/x/vstorage/vstorage.go

```diff
@@ -197,7 +197,11 @@ func (sh vstorageHandler) Receive(cctx *vm.ControllerContext, str string) (ret s
 		entries := make([][]interface{}, len(children.Children))
 		for i, child := range children.Children {
 			entry := keeper.GetEntry(cctx.Context, fmt.Sprintf("%s.%s", path, child))
-			entries[i] = []interface{}{child, entry.Value()}
+			if !entry.HasData() {
```

Comment:
No reason for this method in particular, but it's consistent with the rest of the file because `!entry.HasData()` is often the condition for early exit.